soc/xtensa/intel_adsp/common/adsp.c

```diff
@@ -7,155 +7,10 @@
 //         Rander Wang <rander.wang@intel.com>
 //         Janusz Jankowski <janusz.jankowski@linux.intel.com>
 
-#define RELATIVE_FILE "zephyr/adsp.c"
```

Comment:
I guess there is a wrapper update fro this (to build the file in SOF) or are we good ?

Reply:
I am able to build. I do not know why this was needed.